README.md

```diff
@@ -333,13 +333,17 @@ The recommended way to include __release.sh__ in a project is to:
 4. (Optionally) Create a `.env` file in the `.release` directory filled with
    your upload secrets. (KEY=value pairs each on a new line)
 5. Run __release.sh__.
+6. (Optionally) Running with the `-D` flag will be quicker as it will skip
+   the checkout of external repositories if they are already present,
+   among other shortcuts.
 
 ## Usage
 
 ```text
 Usage: release.sh [options]
   -c               Skip copying files into the package directory.
   -d               Skip uploading.
+  -D               Local dev mode (Skips uploading, keeps existing pkgdir, skips external if it exists)
```

Comment:
Why not just use ./release.sh -de?

Reply:
I'll give it a shot! I don't completely remember why I opted down this path, I did this a few months back, so maybe there was a reason that I'm not recalling right now.

Reply:
I notice I have a branch that performs different git commands and hard-coding to a file-type of alpha when passing in the `-D` flag, but admittedly, I don't recall why I opted for a new flag vs. `-de`

I will say `-D` is different from `-de` in that you don't need to run the first time without `-e` to ensure your externals are present, then run `-de` the second time to avoid fetching again. So it's nice for a `make local` Makefile target in that it runs the same command each time and handles the fetching of externals automagically.

BUT all that to say, that's a pretty "one time" use case and has a fair bit of my own opinionated approach to things, so it's likely not worth adding this specific local flag.